cura/Arranging/ArrangeObjectsJob.py

```diff
@@ -40,6 +40,6 @@ def run(self):
                     i18n_catalog.i18nc("@info:status",
                                        "Unable to find a location within the build volume for all objects"),
                     title = i18n_catalog.i18nc("@info:title", "Can't Find Location"),
-                    message_type = Message.MessageType.WARNING)
+                    message_type = Message.MessageType.ERROR)
```

Comment:
As above; I think it's a warning. The loading /arranging of models will still partially work.